# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/LayersTypes.h
# Commit: 258c6c199656
# Full Hash: 258c6c1996568b3e7d3ca442a2d87df3f60a4b32
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-07 16:20:31
# Regressor Bug: 1464032
# File Overlap Count: 1
# Description:
#   Bug 1464032 Part 6: Add remote canvas pref and refactor TextuteData creation to use it. r=mattwoodrow
#   
#   This is ground work for when we will be returning a recording TextureData for
#   certain types in subsequent patches.
# ==============================================================================

diff -r c83dbcc4dade -r 258c6c199656 gfx/layers/LayersTypes.h
--- a/gfx/layers/LayersTypes.h	Sun Dec 02 14:14:19 2018 +0000
+++ b/gfx/layers/LayersTypes.h	Wed Nov 28 20:44:27 2018 +0000
@@ -155,6 +155,16 @@
   LAYERS_LAST
 };
 
+enum class TextureType : int8_t {
+  Unknown = 0,
+  D3D11,
+  DIB,
+  X11,
+  MacIOSurface,
+  AndroidNativeWindow,
+  Last
+};
+
 enum class BufferMode : int8_t { BUFFER_NONE, BUFFERED };
 
 enum class DrawRegionClip : int8_t { DRAW, NONE };